javascripts/content_scripts/privly.js

```diff
@@ -847,7 +837,7 @@ var privly = {
       "privlybeta\\.org\\/|" + //privlybeta.org
       "localhost:3000\\/" + //localhost:3000
       domainRegexp +
-      ")(\\S){3,}/[^\\s]*\\b","gi");
+      ")(\\S){3,}/[^\\s]*\\b", "gi");
```

Comment:
The `"gi"` should be in the next line. This adds consistency to the fixes done previously at: https://github.com/hkminegod/privly-chrome/blob/master/javascripts/content_scripts/privly.js#L121